public/app/features/expressions/components/Condition.tsx

```diff
@@ -69,65 +69,70 @@ export const Condition: FC<Props> = ({ condition, index, onChange, onRemoveCondi
     condition.evaluator.type === EvalFunction.IsWithinRange || condition.evaluator.type === EvalFunction.IsOutsideRange;
 
   return (
-    <InlineFieldRow>
```

Comment:
💬 I've mostly rearranged some fields in this `Classic Condition` expression to support a narrow card view, the inputs are no longer all one one line but split across multiple lines.